cli/lib/default-config.js

```diff
@@ -4,6 +4,7 @@ module.exports = {
     outDir: "assets",
     noCachePattern: /(\.json)$|(cachebust\.js)/gi,
     cachebustURL: "/cachebust.json",
+    resourcePattern: /(\.js)$|(\.css)$|(\.mjs)$|(\.cjs)$|(\.png)$|(\.jpg)$|(\.gif)$|(\.webp)$|(\.jpeg)$|(\.svg)$/gi,
```

Comment:
Can we also add a required connection default here that would consumed through the files?